win/build.bat

```diff
@@ -0,0 +1,278 @@
+echo Build install directory is [%BUILD_PREFIX%].
+
+if "%DIST_DIR%." == "." set DIST_DIR=%BUILD_DIR%\output
+call :ABSPATH "%DIST_DIR%" DIST_DIR
+echo Packages Output directory is [%DIST_DIR%].
+if NOT EXIST "%DIST_DIR%" md %DIST_DIR%
+GOTO :EOF
+
+
+:BUILD_CUBRID
+echo Building CUBRID in %BUILD_DIR%
+cd /d %BUILD_DIR%
+
+rem TODO: get generator from command line
+if "%BUILD_TARGET%" == "Win32" (
+  set CMAKE_GENERATOR=%BUILD_GENERATOR%
+) else (
+  set CMAKE_GENERATOR="%BUILD_GENERATOR:"=% Win64"
+)
+echo CMAKE_GENERATOR is [%CMAKE_GENERATOR%]
+
+if "%BUILD_GEN_VERSION%" == "V141" (
+  "%CMAKE_PATH%" -G %CMAKE_GENERATOR% -DCMAKE_BUILD_TYPE=%BUILD_TYPE% -DCMAKE_INSTALL_PREFIX=%BUILD_PREFIX% -DPARALLEL_JOBS=10 -DCUBRID_CCI_PACKAGE_NAME=%CUBRID_CCI_PACKAGE_NAME% %SOURCE_DIR%
+) else (
+  "%CMAKE_PATH%" -G %CMAKE_GENERATOR% -DCMAKE_BUILD_TYPE=%BUILD_TYPE% -DCMAKE_INSTALL_PREFIX=%BUILD_PREFIX% -DPARALLEL_JOBS=10 -DCUBRID_CCI_PACKAGE_NAME=%CUBRID_CCI_PACKAGE_NAME% -DFOR_OTHER_DRIVER=true %SOURCE_DIR%
+)
+
+if ERRORLEVEL 1 (echo FAILD. & GOTO :EOF) ELSE echo OK.
+
+"%CMAKE_PATH%" --build . --config %BUILD_TYPE% --target install
+if ERRORLEVEL 1 (echo FAILD. & GOTO :EOF) ELSE echo OK.
+
+GOTO :EOF
+
+
+:BUILD_CCI_PACKAGE
+echo Buiding CCI package in %BUILD_DIR% ...
+if NOT EXIST %BUILD_DIR% echo Cannot found built directory. & GOTO :EOF
+cd /d %BUILD_DIR%
+
+echo drop %CUBRID_CCI_PACKAGE_NAME%.zip into %DIST_DIR%
+"%CPACK_PATH%" -C %BUILD_TYPE% -G ZIP -B "%DIST_DIR%"
+if ERRORLEVEL 1 echo FAILD. & GOTO :EOF
+rmdir /s /q "%DIST_DIR%"\_CPack_Packages
+echo Package created. [%DIST_DIR%\%CUBRID_CCI_PACKAGE_NAME%.zip]
+set DIST_PKGS=%DIST_PKGS% %CUBRID_CCI_PACKAGE_NAME%.zip
+GOTO :EOF
+
+:ABSPATH
+set %2=%~f1
+GOTO :EOF
+
+:FINDEXEC
+if EXIST %3 set %2=%~3
+if NOT EXIST %3 for %%X in (%1) do set FOUNDINPATH=%%~$PATH:X
+if defined FOUNDINPATH set %2=%FOUNDINPATH:"=%
+if NOT defined FOUNDINPATH if NOT EXIST %3 echo Executable [%1] is not found & GOTO :EOF
+call echo Executable [%1] is found at [%%%2%%]
+GOTO :EOF
+
+
+:BUILD_CLEAN
+if "%OPTION_CHECK%" == "false" (
+del /Q /S %BUILD_ROOT_DIR% >nul 2>&1
+rmdir /s /q %BUILD_ROOT_DIR% >nul 2>&1
+del /Q /S %BUILD_ROOT_PREFIX% >nul 2>&1
+rmdir /s /q %BUILD_ROOT_PREFIX% >nul 2>&1
+if EXIST %SOURCE_DIR%\CCI-VERSION-DIST del /Q /S %SOURCE_DIR%\CCI-VERSION-DIST >nul 2>&1
+) else (
+del /Q /S %BUILD_DIR% >nul 2>&1
+rmdir /s /q %BUILD_DIR% >nul 2>&1
+del /Q /S %BUILD_PREFIX% >nul 2>&1
+rmdir /s /q %BUILD_PREFIX% >nul 2>&1
+if EXIST %SOURCE_DIR%\CCI-VERSION-DIST del /Q /S %SOURCE_DIR%\CCI-VERSION-DIST >nul 2>&1
+)
+GOTO :EOF
+
+
+:SHOW_USAGE
+@echo.Usage: build.bat [OPTION] [TARGET]
+@echo.Build and package script for CUBRID CCI (with tools - cci_applier)
+@echo. OPTIONS
+@echo.  /32      or /64    Build 32bit or 64bit applications (default: 64)
+@echo.  /Release or /Debug Build with release or debug mode (default: %BUILD_MODE%)
+@echo.  /vs2017            Build with VS2017 (default: %BUILD_GENERATOR%)
```

Comment:
I think two lines should be modified as follow: (try `build /debug aa`, the you can see **Debug** is default in help msg.)
@echo.  /Release or /Debug Build with release or debug mode (default: **Release**)
@echo.  /vs2017            Build with VS2017 (default: **VS2017**)

Reply:
ok thanks, fixed it.
https://github.com/CUBRID/cubrid-cci/pull/29/commits/c0f07f9089fd02ff3972915115df78efaaab1908

Reply:
> It seems well organized. Now, it looks good to me, thanks for your efforts.
> And one last thing, for consistency, also remove the 'dist' option in build.sh (for Linux).

thanks review, added not supported option process (dist option).
https://github.com/CUBRID/cubrid-cci/pull/29/commits/c0f07f9089fd02ff3972915115df78efaaab1908